care/apps/patients/filters.py

```diff
@@ -11,3 +13,59 @@ class PatientTimelineFilter(filters.FilterSet):
     class Meta:
         model = patients_models.PatientTimeLine
         fields = ("date", "description")
+
+
+class PatientFilter(filters.FilterSet):
+    name = filters.CharFilter(field_name="name", lookup_expr="istartswith")
+    icmr = filters.CharFilter(field_name="icmr_id", lookup_expr="istartswith")
+    govt = filters.CharFilter(field_name="govt_id", lookup_expr="istartswith")
+    facility = filters.CharFilter(field_name="facility")
+    gender = filters.ChoiceFilter(
+        field_name="gender", choices=common_constants.GENDER_CHOICES
+    )
+    years = filters.CharFilter(field_name="year")
+    months = filters.CharFilter(field_name="month")
+    contact = filters.CharFilter(field_name="phone_number", lookup_expr="istartswith")
+    address = filters.CharFilter(field_name="address", lookup_expr="istartswith")
+    district = filters.CharFilter(field_name="district")
+    cluster = filters.CharFilter(field_name="cluster_group")
+    covid_status = filters.CharFilter(field_name="covid_status")
+    patient_status = filters.ChoiceFilter(
```

Comment:
we are showing status from PatientFacility.patient_status.name too. so we need to include both